### ISSUE TITLE
[Feature Request] iOS: Toggle for encrypted DNS

### ISSUE THREAD

**Issue by smolbytes** [blokadaorg/blokada#896]
Thank you for all of the awesome work on this project!

Could you please add a toggle switch on the iOS version of Blockada to turn on and off DoH/DoT, similar to what is available in the Android version? Currently on iOS it is either all or nothing. 

Some people have upstream local DNS they need to rely on, but still want to be able to block trackers and ads on their device. Some corporate environments block outbound connections to popular DoH servers (1.1.1.1, 1.0.0.1, 9.9.9.9, etc) in order to maintain "visibility" of their traffic, but users would still benefit from being able to use Blockada locally on their device even with unencrypted DNS requests. Defaulting to encrypted would still likely be appropriate for most users, but allowing a toggle for those that understand the implications.

**Comment by repunset274** [comment]
I agree, this would be great for my setup with pi-hole at home as well. Blocking on-device and at the network level would be nice double coverage, especially since some of the blocklists are different.
